CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # master
 
 ## breaking changes
+* version v0.12.0-182-ged2adc5b and later move the option `memory-idx.match-cache-size` to the new config section `tag-query.match-cache-size`.
```

Comment:
why can't we keep this in memory-idx, because this option is a setting affecting the behavior of queries on the memory index, no?

Reply:
similar for meta-tag-support, this seems to belong in memory-idx section just like tag-support already is.

Reply:
Simply because that's where they are used when the query expressions get evaluated. It's definitely possible to leave both of them in the memory index, so the setting is there, then the `tagquery` package will just have to import the `idx.memory` package to access it.
Although, I just realized that in the next PR i'd probably have to move the `meta-tag-support` to `idx.memory` anyway, so we might as well do this now.

Reply:
yes, our internal package layout is not a concern that should leak to operators of metrictank